src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+
+  --selected-color: #00acdc;
```

Comment:
It is a common best practice in modern CSS to define global variables in `:root` to ensure that they are universally accessible and clearly signify their global nature.